glossary.md

```diff
@@ -633,6 +626,10 @@ inactive community service role within the Community Facilitators program; commu
 
 initiative in The Carpentries that sought to create new pathways for active involvement in everyday community activities by more community members; this program has been merged into the Community Development Program
 
+#### Community Initiatives
+
+fiscal sponsor of The Carpentries
```

Comment:
Should we add that this was before Carpentries Inc?

Reply:
OH that's a great point. What about: fiscal sponsor of The Carpentries from 2018-2024? Or should we be more accurate with the dates? I can find them...

Reply:
@kariljordan I think the range is good :)